File: dummy_creator.cpp
// includes
#include "dummy_creator.h"
#include "basic_parser/dummy_creator.h"
#include "object_initializer.h"

namespace lang_tools
{

namespace dummy
{

// enums

lang_e create__lang_e()
{
    static const unsigned SIZE = 148;

    static const lang_e values[SIZE] = { lang_e::UNDEF, lang_e::AB, lang_e::AA, lang_e::AF, lang_e::AK, lang_e::SQ, lang_e::AM, lang_e::AR, lang_e::AN, lang_e::HY, lang_e::AS, lang_e::AV, lang_e::AZ, lang_e::BM, lang_e::BA, lang_e::BE, lang_e::BN, lang_e::BH, lang_e::BI, lang_e::BS, lang_e::BR, lang_e::BG, lang_e::MY, lang_e::CA, lang_e::CH, lang_e::CE, lang_e::NY, lang_e::ZH, lang_e::CV, lang_e::KW, lang_e::CO, lang_e::CR, lang_e::HR, lang_e::CS, lang_e::DA, lang_e::DV, lang_e::NL, lang_e::DZ, lang_e::EN, lang_e::ET, lang_e::EE, lang_e::FO, lang_e::FJ, lang_e::FI, lang_e::FR, lang_e::FF, lang_e::GL, lang_e::KA, lang_e::DE, lang_e::EL, lang_e::GN, lang_e::HT, lang_e::HZ, lang_e::HO, lang_e::HU, lang_e::ID, lang_e::GA, lang_e::IG, lang_e::IK, lang_e::IO, lang_e::IS, lang_e::IT, lang_e::JV, lang_e::KL, lang_e::KN, lang_e::KR, lang_e::KK, lang_e::KM, lang_e::KI, lang_e::KY, lang_e::KV, lang_e::KG, lang_e::LB, lang_e::LG, lang_e::LI, lang_e::LN, lang_e::LO, lang_e::LT, lang_e::LUB, lang_e::LV, lang_e::GV, lang_e::MK, lang_e::MG, lang_e::ML, lang_e::MT, lang_e::MI, lang_e::MH, lang_e::MN, lang_e::NA, lang_e::NV, lang_e::ND, lang_e::NE, lang_e::NB, lang_e::NN, lang_e::NO, lang_e::II, lang_e::NR, lang_e::OC, lang_e::OJ, lang_e::OM, lang_e::OR, lang_e::OS, lang_e::PI, lang_e::PL, lang_e::PT, lang_e::QU, lang_e::RM, lang_e::RO, lang_e::RU, lang_e::SC, lang_e::SM, lang_e::SG, lang_e::SR, lang_e::GD, lang_e::SN, lang_e::SI, lang_e::SK, lang_e::SL, lang_e::SO, lang_e::ST, lang_e::ES, lang_e::SU, lang_e::SW, lang_e::SS, lang_e::SV, lang_e::TA, lang_e::TE, lang_e::TH, lang_e::TI, lang_e::BO, lang_e::TK, lang_e::TL, lang_e::TN, lang_e::TO, lang_e::TR, lang_e::TS, lang_e::TY, lang_e::UK, lang_e::VE, lang_e::VI, lang_e::WA, lang_e::CY, lang_e::WO, lang_e::FY, lang_e::XH, lang_e::YO, lang_e::ZA, lang_e::ZU,  };

    auto res = values[ ::basic_parser::dummy::create__uint32() % SIZE ];

    return res;
}

// objects

// messages

} // namespace dummy

} // namespace lang_tools

File: dummy_creator.php
<?php

namespace lang_tools_protocol;


// includes
require_once __DIR__.'/../basic_parser/dummy_creator.php';

// own includes
require_once __DIR__.'/../lang_tools_protocol/object_initializer.php';

// enums

function create_dummy__lang_e()
{
    $SIZE = 148;

    $values = array( lang_e__UNDEF, lang_e__AB, lang_e__AA, lang_e__AF, lang_e__AK, lang_e__SQ, lang_e__AM, lang_e__AR, lang_e__AN, lang_e__HY, lang_e__AS, lang_e__AV, lang_e__AZ, lang_e__BM, lang_e__BA, lang_e__BE, lang_e__BN, lang_e__BH, lang_e__BI, lang_e__BS, lang_e__BR, lang_e__BG, lang_e__MY, lang_e__CA, lang_e__CH, lang_e__CE, lang_e__NY, lang_e__ZH, lang_e__CV, lang_e__KW, lang_e__CO, lang_e__CR, lang_e__HR, lang_e__CS, lang_e__DA, lang_e__DV, lang_e__NL, lang_e__DZ, lang_e__EN, lang_e__ET, lang_e__EE, lang_e__FO, lang_e__FJ, lang_e__FI, lang_e__FR, lang_e__FF, lang_e__GL, lang_e__KA, lang_e__DE, lang_e__EL, lang_e__GN, lang_e__HT, lang_e__HZ, lang_e__HO, lang_e__HU, lang_e__ID, lang_e__GA, lang_e__IG, lang_e__IK, lang_e__IO, lang_e__IS, lang_e__IT, lang_e__JV, lang_e__KL, lang_e__KN, lang_e__KR, lang_e__KK, lang_e__KM, lang_e__KI, lang_e__KY, lang_e__KV, lang_e__KG, lang_e__LB, lang_e__LG, lang_e__LI, lang_e__LN, lang_e__LO, lang_e__LT, lang_e__LUB, lang_e__LV, lang_e__GV, lang_e__MK, lang_e__MG, lang_e__ML, lang_e__MT, lang_e__MI, lang_e__MH, lang_e__MN, lang_e__NA, lang_e__NV, lang_e__ND, lang_e__NE, lang_e__NB, lang_e__NN, lang_e__NO, lang_e__II, lang_e__NR, lang_e__OC, lang_e__OJ, lang_e__OM, lang_e__OR, lang_e__OS, lang_e__PI, lang_e__PL, lang_e__PT, lang_e__QU, lang_e__RM, lang_e__RO, lang_e__RU, lang_e__SC, lang_e__SM, lang_e__SG, lang_e__SR, lang_e__GD, lang_e__SN, lang_e__SI, lang_e__SK, lang_e__SL, lang_e__SO, lang_e__ST, lang_e__ES, lang_e__SU, lang_e__SW, lang_e__SS, lang_e__SV, lang_e__TA, lang_e__TE, lang_e__TH, lang_e__TI, lang_e__BO, lang_e__TK, lang_e__TL, lang_e__TN, lang_e__TO, lang_e__TR, lang_e__TS, lang_e__TY, lang_e__UK, lang_e__VE, lang_e__VI, lang_e__WA, lang_e__CY, lang_e__WO, lang_e__FY, lang_e__XH, lang_e__YO, lang_e__ZA, lang_e__ZU,  );

    $res = $values[ \basic_parser\create_dummy__int32() % $SIZE ];

    return $res;
}

// objects

// messages

// namespace_end lang_tools_protocol


?>
